messaging/messaging/src/main/java/module-info.java

```diff
@@ -22,6 +22,7 @@
 
     requires io.helidon.common.context;
     requires io.helidon.common.configurable;
+    requires io.helidon.common.reactive;
```

Comment:
This module exposes `io.helidon.common.reactive` as part of its API, we should use `requires transitive`. CC @danielkec

Reply:
Fixed